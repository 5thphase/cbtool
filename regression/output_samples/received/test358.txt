 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Sending a termination request for vm_129 (cloud-assigned uuid 4003ED64-71DE-5AC2-B82D-8E18170E06DE)....
 status: Done
 status: Removing all VMC objects attached to this experiment.
 status: VMC 911948F0-FCF1-51EE-AC44-17B9CFF25E5A was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 24EA3917-098B-58C9-AF2C-3E0A59DAC84B was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 791B34C6-B003-5011-886C-4B0CFE0C2355 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 9F436EE3-415E-5648-A215-0B2EF5DB9F7C was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 4552998E-15AF-5AD1-AC15-C261E718E69C was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 9E56E718-83AD-541C-A990-6D386283563B was successfully unregistered on SimCloud "TESTCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Cloud TESTCLOUD was successfully detached from this experiment.
