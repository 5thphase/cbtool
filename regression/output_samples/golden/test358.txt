 status: Waiting for all active AIDRS daemons to finish gracefully....
 status: All AIDRS (daemons and objects were removed).
 status: Removing all VMCRS objects attached to this experiment.
 status: Removing all FIRS objects attached to this experiment.
 status: Removing all AI objects attached to this experiment.
 status: Removing all VM objects attached to this experiment.
 status: Sending a termination request for vm_129 (cloud-assigned uuid 374B32D7-1A6C-52C3-9DE0-40DA7A35F4B2)....
 status: Done
 status: Removing all VMC objects attached to this experiment.
 status: VMC DF9833A7-0821-5093-980B-EF4558C93208 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC B07D97D7-6259-50D3-8A5D-2F6387957881 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 73396447-1A1B-5F78-8FD0-C71D15101114 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 4D841AFA-3629-5865-B277-3C9B87B46AF4 was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC 0B9653CA-E757-5034-A764-F8B41227E0EE was successfully unregistered on SimCloud "TESTCLOUD".
 status: VMC FB6EC147-6C78-5AE6-BD5F-8F8D6A60F5BC was successfully unregistered on SimCloud "TESTCLOUD".
 status: Done
 status: Removing all contents from Object Store (GLOBAL objects,VIEWS, etc.)
Cloud TESTCLOUD was successfully detached from this experiment.
